pyQuARC/schemas/check_messages.json

```diff
@@ -926,5 +942,29 @@
             "url": ""
         },
         "remediation": "Either provide a valid GCMD keyword or submit a request to support@earthdata.nasa.gov to have this keyword added to the GCMD KMS."
+    },
+    "granule_single_date_time_check": {
+        "failure": "The Beginning Date Time and the Ending Date Time are the same.",
+        "help": {
+            "message": "",
+            "url": ""
+        },
+        "remediation": "Recommend providing a Single Date Time instead of a Range Date Time."        
+    },
+    "granule_project_short_name_check": {
+        "failure": "The campaign short name listed in the granule metadata is missing or does not match the campaign short name listed in the collection metadata.",
+        "help": {
+            "message": "",
+            "url": ""
+        },
+        "remediation": "Recommend providing [campaign short name listed in collection metadata] as the campaign short name in the granule metadata."
```

Comment:
Might be worth revisiting this wording later..seems like it might could be confusing